Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: changelog release
+
+SEMTAG="tools/semtag"
+TAG_QUERY="v10.0.1.."
+scope ?= "minor"
+
+changelog-unrelease:
+	git-chglog -o CHANGELOG.md $(TAG_QUERY)
+
+changelog:
+	git-chglog -o CHANGELOG.md --next-tag `$(SEMTAG) final -s $(scope) -o -f` $(TAG_QUERY)
+
```

Comment:
I would like to have the `release` target also in Makefile:
```
release:
	$(SEMTAG) final -s $(scope)
```

Reply:
Sure. Added https://github.com/terraform-aws-modules/terraform-aws-eks/pull/786/commits/6f910751246b9a25d6d3ccd16bf1961fa73ee1c6